src/dao.cpp

```diff
@@ -17,6 +17,15 @@
 
 namespace hypha
 {
+
+   void dao::fix (const checksum256 &hash)
```

Comment:
@lukegravitt - this action is idempotent and permissionless so will not break anything.  When called with an assignment hash, it will add an edge named "role" from the assignment document to the role document.  

If the input hash doesn't exist, isn't an assignment, or the edge pointing to it doesn't exist, the action will fail.  Otherwise it will always pass, including if the edge DOES exist.

So, for migration, we just need to invoke this on every assignment once after we deploy this version.  I have already deployed to the testnet.